tests/test_main.py

```diff
@@ -752,13 +750,15 @@ def prepare(report: List[Dict[str, Any]]):
         # do not use parametrised tests with unittests
         self.maxDiff = 65536
         # instead the config file is used
-        with open(TESTS_PATH / "data" / ".cfg.json", "r") as f:
-            config = json.load(f)
         with tempfile.TemporaryDirectory() as tmp_dir:
-            for cfg in config:
+            for cfg in DATA_TEST_CFG:
                 # important key in .cfg.json is "json_filename"
```

Comment:
It seems it can be removed :)

```suggestion
```

Reply:
applied in https://github.com/Samsung/CredSweeper/pull/360 - the parent of this PR

Reply:
Thank you for a quick fix. I checked.